packages/scandipwa/src/component/ProductDownloadableSamples/ProductDownloadableSamples.style.scss

```diff
@@ -17,7 +17,6 @@
     &-Title {
         font-weight: bold;
         margin-block-end: 5px;
-
```

Comment:
Blank line shouldn't be removed. Usually there should be separation between properties and `@includes`